### DISCUSSION TITLE
School issued email address doesn't get approved

### DISCUSSION THREAD

**Discussion by ghost** [community/community#53007] [GitHub Education]
Hi, I am from the Green University of Bangladesh. My university gives me an email like this: **studnetID@green.ac.bd**. But when I want to go to the next page, it says We require applicants to use a school-issued email address to apply. What is wrong here? And how can I solve this?

**Comment by neutrinomuon** [comment]
Hello @Sazid991, I recommend contacting the support team for assistance. I suggest you launch a ticket over to GitHub Education's Support [here](https://support.github.com/contact/education).

There is also some further info here related to GitHub Global Campus: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student

I hope this helps.
